data/microos/butane/script

```diff
@@ -17,9 +17,22 @@
 #
 set -euxo pipefail
 
+# look for a drive that has no partition table or other information that blkid would return
+detect_free_drive() {
+    declare -a drives
+    drives=( $(ls /sys/block/) )
+    empty=
```

Comment:
Where are we using this variable?

Reply:
No where, that is a left over. Nice catch

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19773